docs/viewports/how-to-add-viewports.md

```diff
@@ -0,0 +1,206 @@
+# How to add viewports
+
+By default, Page Builder defines four viewports, but only two are shown on the Admin stage: `desktop` and `mobile`. The other two viewports (hidden from the stage), are `tablet` and `small-mobile`. This topic shows you how to add the hidden viewports to the stage and customize them as needed.
```

Comment:
I would say it defines four breakpoints and use two of them for viewports

Reply:
Ah, good. I've gone back and forth on the use of viewports vs. breakpoints...good catch.